ext/curl/interface.c

```diff
@@ -1974,6 +1974,9 @@ static void _php_curl_set_default_options(php_curl *ch)
 #endif
 	curl_easy_setopt(ch->cp, CURLOPT_DNS_CACHE_TIMEOUT, 120);
 	curl_easy_setopt(ch->cp, CURLOPT_MAXREDIRS, 20); /* prevent infinite redirects */
+#if !HAVE_IPV6 && LIBCURL_VERSION_NUM >= 0x070a08 /* 7.10.8 */
```

Comment:
We [require libcurl ≥ 7.15.5](https://github.com/php/php-src/blob/php-7.4.26/ext/curl/config.m4#L7) anyway, so this version check is superfluous:
```suggestion
#if !HAVE_IPV6
```